qadence/backends/pyqtorch/backend.py

```diff
@@ -38,6 +39,46 @@
 logger = getLogger(__name__)
 
 
+def set_noise_abstract_to_native(circuit: ConvertedCircuit, config: Configuration) -> None:
```

Comment:
I don't quite get why this is here while `set_noise` is in `transpile`.

Reply:
It's because I return a pyq object, and I'd avoid showing outside pyq objects outside of backends.